### ISSUE TITLE
Error using sageattn varlen

### ISSUE THREAD

**Issue by lestersssss** [kijai/ComfyUI-HunyuanVideoWrapper#290]
got prompt
Loading text encoder model (clipL) from: E:\ComfyUI_lester\ComfyUI\models\clip\clip-vit-large-patch14
Text encoder to dtype: torch.float16
Loading tokenizer (clipL) from: E:\ComfyUI_lester\ComfyUI\models\clip\clip-vit-large-patch14
Loading text encoder model (llm) from: E:\ComfyUI_lester\ComfyUI\models\LLM\llava-llama-3-8b-text-encoder-tokenizer
Loading checkpoint shards: 100%|█████████████████████████████████████████████████████████| 4/4 [00:21<00:00,  5.49s/it]
Text encoder to dtype: torch.float16
Loading tokenizer (llm) from: E:\ComfyUI_lester\ComfyUI\models\LLM\llava-llama-3-8b-text-encoder-tokenizer
HTTP Request: POST https://translate.google.com/_/TranslateWebserverUi/data/batchexecute?rpcids=MkEWBc&bl=boq_translate-webserver_20201207.13_p0&soc-app=1&soc-platform=1&soc-device=1&rt=c "HTTP/2 200 OK"
llm prompt attention_mask shape: torch.Size([1, 161]), masked tokens: 6
clipL prompt attention_mask shape: torch.Size([1, 77]), masked tokens: 6
model_type FLOW
The config attributes {'use_flow_sigmas': True, 'prediction_type': 'flow_prediction'} were passed to FlowMatchDiscreteScheduler, but are not expected and will be ignored. Please verify your scheduler_config.json configuration file.
Scheduler config: FrozenDict({'num_train_timesteps': 1000, 'flow_shift': 9.0, 'reverse': True, 'solver': 'euler', 'n_tokens': None, '_use_default_values': ['num_train_timesteps', 'n_tokens']})
Using accelerate to load and assign model weights to device...
Requested to load HyVideoModel
loaded completely 11603.543690490722 11602.879028320312 False

E:\ComfyUI_lester>pause
请按任意键继续. . .

**Comment by EduSouza-programmer** [comment]
I don't understand, what is the error exactly?
